src/styles/global.scss

```diff
@@ -75,6 +75,21 @@ body {
   }
 }
 
+.demo-section {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+
+  br {
```

Comment:
Out of curiosity, what does the <br> correspond to? I didn't notice any br tags in the demo section.